tests/sentry/api/endpoints/test_organization_index.py

```diff
@@ -151,23 +137,16 @@ def setUp(self):
         self.no_2fa_user = self.create_user()
         self.create_member(organization=self.org_2fa, user=self.no_2fa_user, role="member")
 
-    @fixture
-    def path(self):
-        return reverse("sentry-organization-home", kwargs={"organization_slug": self.org_2fa.slug})
```

Comment:
doesn't the endpoint need to be redefined to use this?

Reply:
The helper `get_success_response` works because first line of OrganizationIndex2faTest is
```
endpoint = "sentry-organization-home"
```
Weirdly `self.path` isn't being used in this class. `self.path_2fa` is defined in `TwoFactorAPITestCase`.